libgeotiff/cmake/project-config.cmake.in

```diff
@@ -3,53 +3,49 @@
 # Set
 #  @PROJECT_NAME@_FOUND = 1
 #  @PROJECT_NAME@_INCLUDE_DIRS = /usr/local/include
-#  @PROJECT_NAME@_SHARED_LIBRARIES = geotiff_library
-#  @PROJECT_NAME@_STATIC_LIBRARIES = geotiff_archive
 #  @PROJECT_NAME@_LIBRARY_DIRS = /usr/local/lib
 #  @PROJECT_NAME@_BINARY_DIRS = /usr/local/bin
 #  @PROJECT_NAME@_VERSION = 1.4.1 (for example)
-#  Depending on @PROJECT_NAME@_USE_STATIC_LIBS
-#    @PROJECT_NAME@_LIBRARIES = ${@PROJECT_NAME@_SHARED_LIBRARIES}, if OFF
-#    @PROJECT_NAME@_LIBRARIES = ${@PROJECT_NAME@_STATIC_LIBRARIES}, if ON
 
 # For compatibility with FindGeoTIFF.cmake, also set
 # @PROJECT_NAME_UPPER@_FOUND
 # @PROJECT_NAME_UPPER@_INCLUDE_DIR
 # @PROJECT_NAME_UPPER@_LIBRARY
 # @PROJECT_NAME_UPPER@_LIBRARIES
 
-message (STATUS "Reading ${CMAKE_CURRENT_LIST_FILE}")
-# @PROJECT_NAME@_VERSION is set by version file
-message (STATUS
-  "@PROJECT_NAME@ configuration, version ${@PROJECT_NAME@_VERSION}")
-
 # Tell the user project where to find our headers and libraries
 get_filename_component (_DIR ${CMAKE_CURRENT_LIST_FILE} PATH)
 get_filename_component (_ROOT "${_DIR}/@PROJECT_ROOT_DIR@" ABSOLUTE)
 set (@PROJECT_NAME@_INCLUDE_DIRS "${_ROOT}/include")
 set (@PROJECT_NAME@_LIBRARY_DIRS "${_ROOT}/lib")
 set (@PROJECT_NAME@_BINARY_DIRS "${_ROOT}/bin")
+unset (_ROOT)
+unset (_DIR)
 
-message (STATUS "  include directory: \${@PROJECT_NAME@_INCLUDE_DIRS}")
-
-if(BUILD_SHARED_LIBS)
-set (@PROJECT_NAME@_SHARED_LIBRARIES @GEOTIFF_LIBRARY_TARGET@)
+set (@PROJECT_NAME@_LIBRARIES @GEOTIFF_LIBRARY_TARGET@)
+if("@BUILD_SHARED_LIBS@")
+  set (@PROJECT_NAME@_SHARED_LIBRARIES ${@PROJECT_NAME@_LIBRARIES})
 else()
-set (@PROJECT_NAME@_STATIC_LIBRARIES @GEOTIFF_LIBRARY_TARGET@)
+  set (@PROJECT_NAME@_STATIC_LIBRARIES ${@PROJECT_NAME@_LIBRARIES})
+  include(CMakeFindDependencyMacro)
+@CONFIG_DEPENDENCIES@
 endif()
-# Read in the exported definition of the library
-include ("${_DIR}/@PROJECT_NAME_LOWER@-depends.cmake")
 
-unset (_ROOT)
-unset (_DIR)
+if(NOT @PROJECT_NAME@_FIND_QUIETLY)
+  message (STATUS "Reading ${CMAKE_CURRENT_LIST_FILE}")
```

Comment:
Is there any use in these messages? They are somewhat annoying because they show up in every single configure run.

Reply:
I didn't invent them. But at least QUIET should have effect now ;-)
Maybe something for the next revision, when we add ZLIB::ZLIB etc.